view-user/src/styles/Home.module.css

```
@@ -10,6 +10,18 @@
   padding: 0.5rem 1rem;
 }
 
+.languageBlock {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  color:#D95B7F;
+  font-size: clamp(3rem, calc(100vw / 20 ) , 5rem);
+}
+
+.languageBlock p {
+  background-color: #07033E;
+}
```

pタグにバックグラウンドをつけてしまうと不便な点が多い
- 背景色の空間に対して文字が大きすぎる
- 不揃え感が出てしまう
- 外側の要素と角丸が合わせられない

背景を塗りつぶすような「空間」を示すようなものはdivタグを使った方が明示的なので、既存のpタグをdivで囲ってそっちに背景をつけるといいかもね
あと、タグ内に`onClick`とかかいてタグ自体が長くなってしまう場合は、タグの開始と終了のインデントを合わせると綺麗になるよ

```
        <div className={styles.languageBlock}>
          <div className={styles.language}>
            <p
              onClick={() => {
                router.push('/', '/', { locale: 'ja' });
                setIsOpened(false);
              }}
            >
              日本語
            </p>
          </div>
          <div className={styles.language}>
            <p
              onClick={() => {
                router.push('/', '/', { locale: 'en' });
                setIsOpened(false);
              }}
            >
              English
            </p>
          </div>
        </div>
```

```suggestion
.language {
  display: flex;
  justify-content: center;
  background-color: #07033E;
  padding: いい感じに;
  border-radius: いい感じに;
}
```

---

レビューありがとうございます！修正しました。